troubadix/plugins/multiple_re_parameters.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2024 Greenbone AG
+
+import re
+from pathlib import Path
+from typing import Iterable, Iterator
+
+from troubadix.helper.patterns import (
+    SpecialScriptTag,
+    get_special_script_tag_pattern,
+)
+from troubadix.plugin import LineContentPlugin, LinterError, LinterResult
+
+re_pattern = re.compile(r"re\s*:")
```

Comment:
```suggestion
RE_PATTERN = re.compile(r"re\s*:")
```
'Cause it's used as a constant

Reply:
amended in 78cfb2fcd7027dc7e3382c0185422fbd3d6fc6c6